frontend/templates/contraceptives/tabs/Efficacy.tsx

```diff
@@ -0,0 +1,88 @@
+import { ReactElement } from 'react';
+import {
+  CategoryValue,
+  Column,
+  Description,
+  HighlightDescription,
+  Row
+} from './StyledComponents';
+import styled from 'styled-components';
+
+// Styles
+
+const AlphaList = styled.ol`
+  color: #7c7c7c;
+`;
+
+const CategoryValueStyled = styled(CategoryValue)`
+  font-size: 1rem;
+`;
+
+const HighlightedDescription = styled(HighlightDescription)`
+  color: black;
+  font-weight: bold;
+`;
+
+// Components
+
+const StopIcons = ({
+  icon,
+  label,
+}: {
+  icon: ReactElement,
+  label: string,
+}) => {
+  return (
+    <>
+      <Column>
+        {icon}
+        <Description>{label}</Description>
```

Comment:
Theres no spacing in-between each description. Let's try to add that in

<img width="515" alt="Screen Shot 2021-11-07 at 7 44 16 PM" src="https://user-images.githubusercontent.com/17476012/140668663-d505cef4-7932-47a2-8bd8-c84942a81c44.png">